packages/rocketchat-livechat/.app/client/lib/commands.js

```diff
@@ -22,26 +23,25 @@ this.Commands = {
 			swal({
 				title: t('Chat_ended'),
 				text: transcriptMessage,
-				type: 'input',
+				input: 'email',
 				inputValue: email,
+				inputPlaceholder: t('Type_your_email'),
 				showCancelButton: true,
 				cancelButtonText: t('no'),
-				confirmButtonText: t('yes'),
-				closeOnCancel: true,
-				closeOnConfirm: false
-			}, (response) => {
-				if ((typeof response === 'boolean') && !response) {
+				confirmButtonText: t('yes')
+			}).then((result) => {		
+				if ((typeof result.value === 'boolean') && !response) {
```

Comment:
since there are a lot of changes here, what do you think on improving it's logic to look more "flat" (as the previous code improvements) ?